web/pdf_viewer.js

```diff
@@ -757,7 +757,8 @@ var PDFViewer = (function pdfViewer() {
       return new AnnotationLayerBuilder({
         pageDiv: pageDiv,
         pdfPage: pdfPage,
-        linkService: this.linkService
+        linkService: this.linkService,
+        downloadManager: new DownloadManager()
```

Comment:
Done in the new commit. Since I'm not too familiar with the components, could you check if this works? I ran `node make generic components` and inspected the code. I see the `DownloadManager` class there now, so I think this will work.